service/controller/app/resource/releasemigration/create.go

```diff
@@ -67,7 +68,11 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	}
 
 	deploy, err := cc.Clients.K8s.K8sClient().AppsV1().Deployments(key.Namespace(cr)).Get(ctx, cr.Name, metav1.GetOptions{})
-	if err != nil {
+	if apierrors.IsNotFound(err) {
```

Comment:
Adding err check logic since It crashed when no chart-operator deployment in the cluster.